scapy/layers/can.py

```diff
@@ -522,5 +522,5 @@ def __exit__(self, exc_type, exc_value, tracback):
     # emulate SuperSocket
     @staticmethod
     def select(sockets, remain=None):
-        # type: (List[SuperSocket], Optional[int]) -> Tuple[List[SuperSocket], None]  # noqa: E501
-        return sockets, None
+        # type: (List[SuperSocket], Optional[int]) -> List[SuperSocket]
```

Comment:
Cool PR. I've seen your refactoring related to PacketList Iterators. I would like to apply the same changes to this CandumpReader.

Reply:
Thanks. I'm unsure if you mean that I should add this change elsewhere ?